addons/mail/static/tests/helpers/mock_server/models/mail_tracking_value.js

```diff
@@ -47,17 +47,17 @@ patch(MockServer.prototype, "mail/models/mail_tracking_value", {
     ) {
         let isTracked = true;
         const irField = this.models["ir.model.fields"].records.find(
-            (field) => field.model === modelName && field.name === fieldName
+            (field_id) => field_id.model === modelName && field_id.name === fieldName
```

Comment:
In this anonymous method field_id is actually field (object and not id), so I would suggest to revert the rename here.